From 0bd591bf27f383dd273d97089040e2c18472f370 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sun, 28 Jul 2024 14:34:37 +0200
Subject: [PATCH] Show only in GNOME

Since currently the MutterActivityMonitor is the only supported idle monitor
backend, this application is not really useful outside of GNOME.
---
 data/applications/org.gnome.BreakTimer.Settings.desktop.in.in | 1 +
 data/autostart/org.gnome.BreakTimer.Daemon.desktop.in.in      | 1 +
 2 files changed, 2 insertions(+)

diff --git a/data/applications/org.gnome.BreakTimer.Settings.desktop.in.in b/data/applications/org.gnome.BreakTimer.Settings.desktop.in.in
index f184b16..aa56b8e 100644
--- a/data/applications/org.gnome.BreakTimer.Settings.desktop.in.in
+++ b/data/applications/org.gnome.BreakTimer.Settings.desktop.in.in
@@ -8,5 +8,6 @@ Categories=GNOME;GTK;Utility;
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=@APPLICATION_ID@
 Exec=gnome-break-timer-settings
+OnlyShowIn=GNOME;
 StartupNotify=true
 DBusActivatable=true
diff --git a/data/autostart/org.gnome.BreakTimer.Daemon.desktop.in.in b/data/autostart/org.gnome.BreakTimer.Daemon.desktop.in.in
index 7145163..6fdaede 100644
--- a/data/autostart/org.gnome.BreakTimer.Daemon.desktop.in.in
+++ b/data/autostart/org.gnome.BreakTimer.Daemon.desktop.in.in
@@ -6,6 +6,7 @@ Comment=Computer break reminders for GNOME
 Icon=@APPLICATION_ID@
 NoDisplay=true
 Exec=gnome-break-timer-daemon
+OnlyShowIn=GNOME;
 AutostartCondition=GSettings org.gnome.BreakTimer enabled
 X-GNOME-Autostart-Phase=Applications
 X-GNOME-Autostart-Delay=30
-- 
GitLab

